app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -66,7 +66,7 @@ const usestyles = makeStyles((theme) => ({
     flexDirection: "row",
     borderStyle: "solid",
     overflow: "hidden",
-    backgroundColor: palette.white,
+    backgroundColor: theme.mode == "light" ? theme.colors.white : theme.colors.grey10,
```

Comment:
so we don't want to really do this pattern. the idea is that if we have theme.colors.white, then this is the value for the light mode, and for darkMode the value should be translate accordingly. (black in this scenario).

There is some exception here because in the darkMode we have basically a dark background, and in the lightNode, we have sometime white, sometime light grey background. 

we should see the fact the background is sometime lightGrey instead of white for the lightMode as an intermediary state.

Those screens should eventually be refactored to be on par with the darkMode so we don't need to tweak those values per (light/dark) mode

the way to deal with that in the meantime is to create intermediary colors that don't follow the "colors should be inverted between light and dark mode" type of pattern. I have some example of this hack with `lapisLazuliOrLightGrey` or `whiteOrDarkGrey`

Reply:
Understood, implemented the change!